src/modint.rs

```diff
@@ -1 +1,747 @@
+//! Structs that treat the modular arithmetic.
+//!
+//! # Major changes from the original ACL
+//!
+//! - Converted the struct names to PascalCase.
+//! - Renamed `mod` → `modulus`.
+//! - Moduluses are `u32`, not `i32`.
```

Comment:
Sorry for being a little pedantic, but...
```suggestion
//! - Moduli are `u32`, not `i32`.
```

Reply:
I’d forgotten. thank you